wezterm-gui/src/termwindow/mouseevent.rs

```diff
@@ -165,7 +192,7 @@ impl super::TermWindow {
         if let Some(item) = ui_item {
             self.mouse_event_ui_item(item, pane, y, event, context);
         } else {
-            self.mouse_event_terminal(pane, x, y, event, context);
+            self.mouse_event_terminal(pane, x, y, x_pixel_offset, y_pixel_offset, event, context);
```

Comment:
how bad is the fan out from passing `ClickPosition` down here in place of the separate coordinates?  Does it add much value?

Reply:
No extra value that I can see.  I can switch these to ClickPosition also.

Reply:
mouse_event_terminal isn't referenced anywhere else.  I can also make it not pub. ?